bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/RowMerger.java

```diff
@@ -409,6 +436,43 @@ private FlatRow buildRow() {
    */
   public RowMerger(StreamObserver<FlatRow> observer) {
     this.observer = observer;
+    rescheduleNextRowTask();
+  }
+
+  /** Resets the timer for waiting on next row. Will be called when a row is emitted. */
+  private void rescheduleNextRowTask() {
+    if (nextRowFuture != null) {
+      nextRowFuture.cancel(false);
+    }
+    lastRowEmittedAtMillis = System.currentTimeMillis();
+    nextRowFuture =
+        NEXT_ROW_MONITOR_EXECUTOR_SERVICE.scheduleAtFixedRate(
+            () -> {
```

Comment:
please switch this to `new Runnable(...` and revert the compile source change

Reply:
done.